CMakeLists.txt

```diff
@@ -201,14 +201,19 @@ IF(WANT_LV2)
 	IF(PKG_CONFIG_FOUND)
 		PKG_CHECK_MODULES(LV2 lv2)
 		PKG_CHECK_MODULES(LILV lilv-0)
+	ENDIF()
+	IF(NOT LV2_FOUND AND NOT LILV_FOUND)
```

Comment:
Just wanted to mention an unusual scenario: what if `lv2` installation provides pkg-config file only whereas `lilv` provides CMake config file only? I know that won't happen for normal users, so please feel free to ignore.

Reply:
I considered that, but didn't think it was worth handling. Also, we probably want to fail in that situation anyway. `lilv` should depend on `lv2`, so there would be a risk of having headers for two different versions of `lv2` in the include path.

Reply:
> `lilv` should depend on `lv2`, so there would be a risk of having headers for two different versions of `lv2` in the include path.

Makes sense.

Reply:
Thanks, I checked Ubuntu 18.04 as well as Homebrew and can confirm that both ship pkg-config files, neither ship cmake files. Thanks for the explanation.